pyiceberg/catalog/hive.py

```diff
@@ -389,8 +389,8 @@ def _create_hive_table(self, open_client: Client, hive_table: HiveTable) -> None
 
     def _get_hive_table(self, open_client: Client, database_name: str, table_name: str) -> HiveTable:
         try:
-            return open_client.get_table(dbname=database_name, tbl_name=table_name)
-        except NoSuchObjectException as e:
+            return open_client.get_tables(db_name=database_name, pattern=table_name).pop()
```

Comment:
It looks like `get_tables` is available in previous versions of thrift clients 

i see
```
    def get_tables(self, db_name, pattern):
        """
        Parameters:
         - db_name
         - pattern

        """
        self.send_get_tables(db_name, pattern)
        return self.recv_get_tables()
```
in `vendor/hive_metastore/ThriftHiveMetastore.py`